utils/mmkv.ts

```diff
@@ -11,6 +11,29 @@ const storage = new MMKV();
 
 export default storage;
 
+const authMMKV = new MMKV({ id: "converse-auth" });
+
+type AuthStorageKey = "CONVERSE_REFRESH_TOKEN" | "CONVERSE_ACCESS_TOKEN";
+
+export const authMMKVStorage = {
```

Comment:
This should be removed

Reply:
nice catch - leftovers from git problems